### PR TITLE
Add transition none style to Popper to avoid unwanted possible transitions set by sites #5098

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5098

### CODE REVIEW

**Comment by rart** [comment]
Please explain

**Comment by jvega190** [comment]
@rart if there's a transition style set to a `role="tooltip"` element, it will apply to our `ZoneMarkers`, which I don't think we want. For example look at https://www.loom.com/share/5dfcb828862448ebb5305fee56f6b244

File: ui/guest/src/react/ZoneMarker.tsx [hunk 1]
```diff
@@ -89,6 +89,9 @@ export function ZoneMarker(props: ZoneMarkerProps) {
           anchorEl={elRef.current}
           placement="top-start"
           onClick={onPopperClick}
+          style={{
+            transition: 'none'
+          }}
```
Comment (rart) [inline ui/guest/src/react/ZoneMarker.tsx]: Add it to the sx and type it appropriately.

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]
@jvega190 popper does have a transition prop. does that not serve the purpose?

**Review by rart — APPROVED** [review]

**Pull request merged**